### PR TITLE
Feature/basic list

### PR DESCRIPTION
**Pull request by iam-Akshat**
- Use local storage for offline data storage
- Use ES6 class syntax to create objects for Project and Todo
- Use arrow functions for better readability
- Add ability to add Projects
- Add ability to create todo,update and delete them

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light:

Hi @iam-Akshat and @ahmadchata, congratulations on this project :muscle: The following requirements have been met:
- Application 
  - DOM Manipulation and logic are separated into modules :100: 
  - Webpack implemented :+1: 
  - LocalStorage well done :1st_place_medal: 
- Interface
  - User can view all projects :+1: 
  - User can view all to-dos in each project :100: 
  - User can edit and delete a to-do
- Projects
  - We can have different projects
  - User can create a new project :+1: 
  - There is a default project :+1: 
  - User can assign a todo list to a project :1st_place_medal: 
- Todo item is an object with all the required variables

Before we proceed and get our project approved, let us kindly address the comments below:

- Kindly implement the Module Pattern. We use the Module Pattern for reuseability and maintainability
- Let us also implement the factory function. Instead of using a Class or a Constructor. Factory functions return objects without the `new` keyword. Mostly used as API wrapper
- Kindly remove the `console.log` statements in our code, it makes our code ugly and difficult to read
- Kindly add a professional README file to our project. Professional Readme files help both technical and non-technical people to appreciate and interact with our app

Once these are addressed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/views/projectsNav.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line no-console
+const renderProjectsNav = (projectList, onClickHandler = () => { console.log('Add onclick handler function'); }) => {
```
Comment (Forison) [inline src/views/projectsNav.js]: Let us remove `console.log`

**Review by Forison — CHANGES_REQUESTED** [review]
# Status:: Changes required❌

## Good job team😇, 

#### The code is clean and well organized into module, re-usability and maintainability of the code base is quite solid, however, there are two outstanding recommendations that we have to revisit🤗.

* [ ] Let us get rid of `console.log` from the code, we don't want to expose information of our code in production, let us  use `console.log` in development and avoid to commit them in production.

* [ ]  The readme has to be very professional, thus taking into consideration that not all visitors would be technical people. [Checkout microverse template of a readme](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
. Let us  improve the readme by adding:
- Link of live demo of application
- A short description of the aplication
- Image of the application
- Let outline all the steps explicitly as to how to set up and launch the application on your local machine.


Check my comments below👇

#### That is it, kindly address the changes and submit a re-review💪

#### Happy coding

Addo Forison

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**